kuwala/core/canvas/src/data/state/models/CanvasModel.js

```diff
@@ -452,4 +566,10 @@ const getLabelByDataCatalogId = (catalogId) => {
     }
 }
 
+const getElementById = (elements, elementId) => {
+    const selected = elements.filter((el) => el.id === elementId);
+    if (selected.length === 1) return selected[0];
+    return null;
```

Comment:
A more consice and performant way would by this:

```js
try {
  return elements[elements.findIndex(el => el.id === elementId)]
} catch (error) {
  console.error(`Couldn't find element with id ${elementId}`)
}
```

Reply:
Resolved by using `find` instead of `findIndex`

https://stackoverflow.com/questions/10457264/how-to-find-first-element-of-array-matching-a-boolean-condition-in-javascript